tar/write.c

```diff
@@ -1102,6 +1102,7 @@ write_entry_backend(struct bsdtar *bsdtar, struct archive *a,
 			    archive_error_string(a));
 			exit(1);
 		}
+		siginfo_skipped(bsdtar, skiplen);
```

Comment:
Is this the only place we call `archive_write_skip`?

Is there some way we can make `archive_write_skip` adjust the relevant value within libarchive so that we get the right value out from `archive_position_uncompressed`?

Reply:
oops.  Si, we call `archive_write_skip` elsewhere.

My second instinct is to add `siginfo_skipped` at the relevant places because it's not immediately obvious whether messing with `archive.file_position` might break something in libarchive.

Reply:
> My second instinct is to add siginfo_skipped at the relevant places because it's not immediately obvious whether messing with archive.file_position might break something in libarchive.

You know, we have the source code... where does `file_position` get used in libarchive when we're writing an archive?